src/python/antares_xpansion/driver.py

```diff
@@ -51,10 +51,12 @@ def __init__(self, config_loader: ConfigLoader):
             SolversExe(
                 self.config_loader.benders_exe(),
                 self.config_loader.merge_mps_exe(),
+                self.config_loader.presolve_exe(),
                 self.config_loader.outer_loop_exe()),
             self.config_loader.options_file_name(),
             self.config_loader.mpi_exe(),
         )
+        # TODO options.json is empty at this stage. How to force the solver to be Xpress?
```

Comment:
I think we need to comply with the user's choice in the settings.ini. Maybe notice early from the Python orchestrator if the presolve part is called without Xpress, and raise an exception in this case ?

Reply:
Hm... you are suggesting that it's the job of the Python orchestrator to verify the compliance of the solver? I can look how to do it